packages/chopsticks/src/cli.ts

```diff
@@ -81,6 +82,33 @@ yargs(hideBin(process.argv))
       await runBlock(await processArgv(argv))
     }
   )
+yargs(hideBin(process.argv))
+  .scriptName('chopsticks')
+  .command(
+    'try-runtime',
+    'Runs runtime upgrade',
+    (yargs) =>
+      yargs.options({
+        ...defaultOptions,
```

Comment:
we need to make wasm-override required for this command

Reply:
i am not sure about it . in theory i can add data to chain and try runtime. without chaning wasm.

Reply:
But you need to provide a wasm build with feature "try-runtime" right?